src/main/java/org/kiwiproject/test/junit/jupiter/MongoServerExtension.java

```diff
@@ -8,8 +8,10 @@
 import com.mongodb.MongoClientURI;
 import com.mongodb.client.MongoDatabase;
 import de.bwaldvogel.mongo.MongoServer;
+import de.bwaldvogel.mongo.ServerVersion;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
+import org.h2.tools.Server;
```

Comment:
I am guessing this H2 `Server` import was accidentally added and should be removed?